<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fpjc</class>
 <widget class="QDialog" name="fpjc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/pic/翻抛系统背景_3.jpg);</string>
  </property>
  <widget class="FpjWatchs" name="fpjWatchsWidget" native="true">
   <property name="geometry">
    <rect>
     <x>691</x>
     <y>138</y>
     <width>672</width>
     <height>660</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent; </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>1810</x>
     <y>418</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>压力</string>
   </property>
  </widget>
  <widget class="yhcDial" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>398</x>
     <y>744</y>
     <width>230</width>
     <height>230</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent; </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>1810</x>
     <y>148</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>压力</string>
   </property>
  </widget>
  <widget class="QPushButton" name="alertButton">
   <property name="geometry">
    <rect>
     <x>1540</x>
     <y>764</y>
     <width>238</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
background-image: url(:/pic/报警信息-拷贝.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>1781</x>
     <y>152</y>
     <width>21</width>
     <height>11</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/pic/形状-1-拷贝-2.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="ZsylChart" name="leftCurveWidget" native="true">
   <property name="geometry">
    <rect>
     <x>1427</x>
     <y>188</y>
     <width>440</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent; </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>1690</x>
     <y>422</y>
     <width>21</width>
     <height>11</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/pic/形状5.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="yhUpButton">
   <property name="geometry">
    <rect>
     <x>279</x>
     <y>942</y>
     <width>70</width>
     <height>64</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>868</y>
     <width>250</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(26, 216, 191);</string>
   </property>
   <property name="text">
    <string>右履带编码器</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>1720</x>
     <y>148</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>转速</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>1781</x>
     <y>422</y>
     <width>21</width>
     <height>11</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/pic/形状-4.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="speedDownButton">
   <property name="geometry">
    <rect>
     <x>87</x>
     <y>768</y>
     <width>70</width>
     <height>62</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="yhDownButton">
   <property name="geometry">
    <rect>
     <x>87</x>
     <y>943</y>
     <width>70</width>
     <height>62</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>837</x>
     <y>11</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>35</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
color:rgb(200, 200, 200);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>上料机控制</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="indent">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="yhChoiseLabel">
   <property name="geometry">
    <rect>
     <x>156</x>
     <y>943</y>
     <width>124</width>
     <height>62</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>50%</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>817</x>
     <y>68</y>
     <width>320</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color: rgb(50, 150, 100);</string>
   </property>
   <property name="text">
    <string>FLIP CONTROL</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="historyButton">
   <property name="geometry">
    <rect>
     <x>1540</x>
     <y>853</y>
     <width>238</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
background-image: url(:/pic/历史记录-拷贝.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="ZsylChart" name="rightCurveWidget" native="true">
   <property name="geometry">
    <rect>
     <x>1427</x>
     <y>458</y>
     <width>440</width>
     <height>190</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent; </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>1690</x>
     <y>152</y>
     <width>21</width>
     <height>11</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/pic/形状-1-拷贝.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>1739</x>
     <y>18</y>
     <width>130</width>
     <height>62</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>1720</x>
     <y>418</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>转速</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>698</y>
     <width>250</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(26, 216, 191);</string>
   </property>
   <property name="text">
    <string>左履带编码器</string>
   </property>
  </widget>
  <widget class="QLabel" name="speedLabel">
   <property name="geometry">
    <rect>
     <x>156</x>
     <y>768</y>
     <width>124</width>
     <height>62</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>50%</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="speedUpButton">
   <property name="geometry">
    <rect>
     <x>279</x>
     <y>767</y>
     <width>70</width>
     <height>64</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="wlanLabel">
   <property name="geometry">
    <rect>
     <x>1577</x>
     <y>38</y>
     <width>32</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
background-position:center;
background-repeat:no-repeat;
background-image: url(:/pic/信号4.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="assistButton">
   <property name="geometry">
    <rect>
     <x>1540</x>
     <y>940</y>
     <width>238</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
background-image: url(:/pic/辅助设备-拷贝.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="leftFpButton">
   <property name="geometry">
    <rect>
     <x>1529</x>
     <y>135</y>
     <width>100</width>
     <height>42</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pic/左翻抛.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="leftLdButton">
   <property name="geometry">
    <rect>
     <x>1430</x>
     <y>135</y>
     <width>100</width>
     <height>42</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pic/左履带.png) 0 0 0 100;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="rightFpButton">
   <property name="geometry">
    <rect>
     <x>1529</x>
     <y>405</y>
     <width>100</width>
     <height>42</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pic/右翻抛.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="rightLdButton">
   <property name="geometry">
    <rect>
     <x>1430</x>
     <y>405</y>
     <width>100</width>
     <height>42</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pic/右履带.png) 0 0 0 100;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="test_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>0</y>
     <width>200</width>
     <height>62</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pic/hq_logo.png);
background: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>620</width>
     <height>510</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/pic/fpj.png);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <widget class="QLabel" name="SLJ_LeftFront_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>216</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="SLJ_LeftBehind_label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>340</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>216</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="SLJ_RightFront_label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>160</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>216</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="SLJ_RightBehind_label">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>240</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>216</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="SLJ_Left_Roller_Speed_label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>380</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>216</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="SLJ_Right_Roller_Speed_label">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>310</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>216</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="ControlHistory" name="hisWidget" native="true">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>809</y>
     <width>690</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ZsylChart</class>
   <extends>QWidget</extends>
   <header>zsylchart.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>yhcDial</class>
   <extends>QWidget</extends>
   <header>yhcdial.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ControlHistory</class>
   <extends>QWidget</extends>
   <header>controlhistory.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FpjWatchs</class>
   <extends>QWidget</extends>
   <header location="global">fpjwatchs.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
